src/__tests__/login.spec.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { render, fireEvent, cleanup, waitFor } from '@testing-library/react';
+import Login from '../views/Login.View';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import userEvent from '@testing-library/user-event';
+
+const Component = render(
+	<Provider store={store}>
+		<Login />
+	</Provider>
+);
+describe('LOGIN', () => {
+	afterEach(cleanup);
+	it('should render login component', () => {
+		const { asFragment } = render(
+			<Provider store={store}>
+				<Login />
+			</Provider>
+		);
```

Comment:
Use the `Component` variable as you have declared above

Reply:
Changed